code/__HELPERS/radiation.dm

```diff
@@ -27,6 +27,10 @@
 /proc/radiation_pulse(atom/source, intensity, range_modifier, log=FALSE, can_contaminate=TRUE)
 	if(!SSradiation.can_fire)
 		return
+	if(istype(get_turf(source), /turf/open/pool))
+		var/turf/open/pool/PL = get_turf(source)
```

Comment:
Let's get_turf once here (there's another used for logging at the bottom as well!)